garcon/activity.py

```diff
@@ -253,7 +253,16 @@ def run(self):
         previous activity is consumed (context).
         """
 
-        activity_task = self.poll()
+        try:
+            activity_task = self.poll()
+        except Exception as error:
```

Comment:
Can we replace `Exception` with `boto.exception.SWFResponseError`? It's more specific.